test/unit/vl-wizard.test.html

```diff
@@ -100,6 +100,24 @@ <h3 slot="title">Stap 2</h3>
           done();
         });
       });
+
+      test('er kan programmatisch genavigeerd worden naar de vorige/volgende pagina', (done) => {
+        const wizard = fixture('vl-wizard-fixture');
+        setTimeout(() => {
+          let pane = wizard._activePane;
+          sandbox.spy(pane, 'next');
+          sandbox.spy(pane, 'previous');
+          assert.isFalse(pane.next.called);
+          assert.isFalse(pane.previous.called);
+          wizard.next();
+          pane = wizard._activePane;
+          assert.isTrue(pane.next.called);
+          assert.isFalse(pane.previous.called);
+          wizard.previous();
+          assert.isTrue(pane.previous.called);
+          done();
```

Comment:
dat zou ik hier niet testen door te spy'en, maar door effectief in de wizard te kijken dat het volgende/vorige panel actief is ...

Reply:
Het probleem is dat dit door het onderliggende framework gebeurt en we dat eigenlijk niet willen mee testen, de functionele navigatie werking wordt wel e2e gecontroleerd in de e2e testen.

Reply:
ik zie geen wijzigingen in de e2e tests in deze PR ...?

Reply:
De navigatie e2e testen bestonden al, dit is puur een programmatische uitbreiding, de unit test dekt normaal gezien voldoende. Hiervoor nog eens een e2e test voorzien is onnodig aangezien er geen gebruiker aan te pas komt.

Reply:
ja, dan zou ik persoonlijk niet spy'en in de unit tests.

Reply:
In de code gebeurt bij een `next` en `previous` alleen een click op het bijhorende slot element. Dit is wat ik in unit ook test door de click op de slot elementen te controleren. De click niet via spy testen en laten doorvoeren gaat niet veel extra toevoegen aan de test?